src/aosm/azext_aosm/deploy/deploy_with_arm.py

```diff
@@ -164,23 +116,30 @@ def deploy_nfd_from_bicep(
             )
             print(message)
             logger.info(message)
-            self.deploy_bicep_template(bicep_path, parameters)
+            logger.debug(
+                "Parameters used for NF definition bicep deployment: %s", self.parameters
+            )
+
+            self.deploy_bicep_template(bicep_path, self.parameters)
```

Comment:
Don't need to pass in self.parameters now, deploy_bicep_template can just use self.parameters (in fact validate_and_deploy_arm_template can instead I think

Reply:
I didn't do this because I think `deploy_bicep_template()` is used for both manifests and nfd/nsd templates. The former have their own parameter.

I suspect there is a neater way of handling this (probably checking `resource_type` in that method, but I think I've left it too late for me to safely do that now :-(

What do you think?

Reply:
Agreed, I hadn't grasped that.